crates/walrus-service/src/client/daemon/routes.rs

```diff
@@ -911,4 +1129,37 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_quilt_file_metadata_deserialization() {
+        let json = r#"[
+            {
+                "identifier": "contract-v2",
+                "tags": {
+                    "author": "alice",
+                    "version": "2.0"
+                }
+            },
+            {
+                "identifier": "logo-2024",
+                "tags": {
+                    "type": "logo",
+                    "format": "png"
```

Comment:
What happens when an integer or float or array value is there on the right-hand side? Will that error out?